ansible_collection/hpe/nimble/plugins/modules/hpe_nimble_chap_user.py

```diff
@@ -144,7 +143,7 @@ def update_chap_user(
 
         changed_attrs_dict, params = utils.remove_unchanged_or_null_args(user_resp, **kwargs)
         if changed_attrs_dict.__len__() > 0:
-            user_resp = client_obj.chap_users.update(id=user_resp.attrs.get("id"), password=user_password, **params)
+            user_resp = client_obj.chap_users.update(id=user_resp.attrs.get("id"), **params)
```

Comment:
as per the text description of "iscsi_initiators" of CHAP in REST doc. we should not pass password in case we want to update iscsi. hence making it as optional param rather than passing it always.